src/devices/MAX31856/MAX31856.cs

```diff
@@ -46,12 +46,12 @@ public bool TryGetTemperature(out Temperature temperature)
             try
             {
                 temperature = ReadTemperature(out byte[] spiOutputData);
-                return true;
+                return false;
```

Comment:
that's not what I meant when I said "Reverse" I meant to avoid try..catch at all by not throwing and having TryGetTemperature method doing the communication and ReadTemperature method calling into TryReadTemperature and throwing based on what it returns.

The previous `true`/`false` behavior was correct (true on successful reads)

Reply:
I attempted to fix this but am not sure if I am doing it correctly. Sorry Im new to C# and doing my best to learn it! (Mechanical engineering background).

I am also getting this error all of a sudden in VS 2019:
Error	NU1007	The runtime.json specified in the project 'C:\Program Files\dotnet\sdk\5.0.100\RuntimeIdentifierGraph.json' could not be found.	System.Device.Gpio	C:\Dev\iot2\src\System.Device.Gpio\System.Device.Gpio.csproj

I couldn't figure out a way to fix this.

Reply:
actually we can just create a single `Temperature GetTemperature()` method which throws on error, no other `TryGetTemperature` etc.  that will simplify it

Reply:
Ok did that makese sense